realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsQueryTests.java

```diff
@@ -393,13 +392,14 @@ public void isEmpty_acrossLinkingObjectObjectLink() {
     @Test
     public void isEmpty_acrossLinkingObjectListLink() {
         createIsEmptyDataSet(realm);
+        assertEquals(2, realm.where(AllJavaTypes.class).findAll().size());
         for (RealmFieldType type : SUPPORTED_IS_EMPTY_TYPES) {
             switch (type) {
                 case STRING:
-                    assertEquals(1, realm.where(AllJavaTypes.class).isEmpty(AllJavaTypes.FIELD_LO_LIST + "." + AllJavaTypes.FIELD_STRING).count());
+                    assertEquals(0, realm.where(AllJavaTypes.class).isEmpty(AllJavaTypes.FIELD_LO_LIST + "." + AllJavaTypes.FIELD_STRING).count());
```

Comment:
I suspect that this test is not very robust.  There are so many ways that the query could fail to return any entries.  Is there any way to fiddle the data so that there is one thing there?